action.yml

```diff
@@ -18,11 +18,32 @@ inputs:
   parent:
     description: 'If specified, the branch will be reset to the parent branch'
     required: false
+  cs_username:
```

Comment:
why not more traditional "role name"?

Reply:
renamed cs_username -> cs_role_name.
If you asking why there is cs_ prefix - it's to understand that all those parameters are needed for cs subcommand or for connection string. 
In create-branch-action those parameters are without prefix and I was misled couple of times, thinking that they are used to create the branch, but no - same as here - they are needed only to get the connection string.